packages/blocks/src/render-blocks.js

```diff
@@ -62,14 +62,34 @@ const parseNodes = ( nodes ) =>
 			name: toLower( node.tagName ),
 			children: parseNodes( node.childNodes ),
 			props: {
-				...attributes,
+				...parseTagProps( toLower( node.tagName ), attributes ),
```

Comment:
Don't really need `toLower` here - `node.tagName` is guaranteed to be all caps here and it's hard to even say how much of a convenience it is to have it in lowercase in the switch statement below since the value isn't used anywhere else?